inc/config.php

```diff
@@ -254,6 +254,13 @@
                         $newer = true;
                      }
                   }
+               } else if (strlen($CFG_GLPI['dbversion']) > 40) {
```

Comment:
`preg_match('/^.+@[0-9a-f]{40}$/i', $CFG_GLPI['dbversion'])` could be more secure, but I do not think we will have a 40+ stable version string length someday.